### DISCUSSION TITLE
Lint for syntax-quoted symbols with a dot

### DISCUSSION THREAD

**Discussion by vemv** [clj-kondo/clj-kondo#1479] [Ideas]
One of these things is not like the other:

```clj
`a
`a-a
`a.a ;; <-------
::a
::a-a
::a.a
```

Its lack of ns-expansion is something I only recently learned about (even after sprinkling dots in ns-qualified things for a number of years).

So it seems a deceiving construct, which can lead to buggy code.

What would you think about linting for them?

Cheers - V

**Comment by borkdude** [comment]
Can you please fill out the issue template when posting a next issue? 

What do you find deceiving? 

Using symbols with dots in syntax-quote is a pretty common phenomenon. I have several dozens of them here: https://github.com/babashka/babashka/blob/26336ed5006ebfbc127da9eaf3d522d07aaf6e15/src/babashka/impl/classes.clj#L99

So I don't think warning about them, without any more information about your issue, which the issue template kind of forces you to provide, is a good idea.

**Comment by borkdude** [comment]
Converted to a discussion so with more info, an actionable issue could be created.
